macosx/Controller.m

```diff
@@ -234,8 +234,51 @@ static void removeKeRangerRansomware()
 }
 
 @implementation Controller
+{
+    tr_session                      * fLib;
+
+    NSMutableArray                  * fTorrents, * fDisplayedTorrents;
+
+    PrefsController                 * fPrefsController;
+    InfoWindowController            * fInfoController;
+    MessageWindowController         * fMessageController;
+
+    NSUserDefaults                  * fDefaults;
+
+    NSString                        * fConfigDirectory;
+
+    DragOverlayWindow               * fOverlayWindow;
+
+    io_connect_t                    fRootPort;
+    NSTimer                         * fTimer;
+
+    VDKQueue                        * fFileWatcherQueue;
+
+    StatusBarController             * fStatusBar;
+
+    FilterBarController             * fFilterBar;
+
+    QLPreviewPanel                  * fPreviewPanel;
+    BOOL                            fQuitting;
+    BOOL                            fQuitRequested;
+    BOOL                            fPauseOnLaunch;
+
+    Badger                          * fBadger;
+
+    NSMutableArray                  * fAutoImportedNames;
+    NSTimer                         * fAutoImportTimer;
+
+    NSMutableDictionary             * fPendingTorrentDownloads;
+
+    NSMutableSet                    * fAddingTransfers;
+
+    NSMutableSet                    * fAddWindows;
+    URLSheetWindowController        * fUrlSheetController;
+
+    BOOL                            fGlobalPopoverShown;
+    BOOL                            fSoundPlaying;
+}
 
-#warning remove ivars in header when 64-bit only (or it compiles in 32-bit mode)
 @synthesize prefsController = fPrefsController;
```

Comment:
I think the minimum OS doesn't support 32-bit anymore, so these can be removed.

Reply:
Do you want to replace this with implicit getter? I thought synthesize in this case just a shorthand syntax for wiring getter/setter of property with declared field.

Reply:
I think you can just remove those and it will just work. Of course a more modern codebase would use properties and not variables directly, but for now I think just remove the synthesize should work.

Reply:
It won't work as this properties are not wired with anything and will return `nil` on access.
For example there https://github.com/transmission/transmission/blob/master/macosx/PrefsController.m#L109
```objc
[(Controller *)[NSApp delegate] fileWatcherQueue]
```
will return `nil` and this code will be no-op.

Reply:
I'm a bit confused - why would removing the _@synthesize_s result in nil?

Reply:
Properties in `@interface` are synthesised by default with ivar starting with underscore.
If we declare `@preperty Type prop`, we can access it's backing field by `self._prop`.
We can rename this field or use already existing field by `@synthesize propName = fieldName`.
In our case we already have field `fFileWatcherQueue`.
If we remove `@synthesize ...` like above, property `fileWatcherQueue` and field `fFileWatcherQueue` won't point to each other.
Thus `fileWatcherQueue` will be `nil`

Reply:
Oh sorry I got variables starting with `f` confused with the corresponding property that doesn't have `f`. The proper solution is the use the properties directly and not refer to the backing variable.